### ISSUE TITLE
can't find the Songs Path

### ISSUE THREAD

**Issue by Golgrax** [MrHellYea/Py-OsuAuto#9]
well I'm a Linux Ubuntu user, new to Osu! and made some changes in the code.
what my problem is i can't find the Songs folder. I tried using the Beatmap editor and saw what I made with .olz format

![my screenshot](https://github.com/user-attachments/assets/f21b1e6d-77a2-4df5-9058-bb83238a4ca6)

**Comment by vicb0** [comment]
The songs folder is in your Osu's directory, not in the bot's directory. It's where the game stores the beatmaps

**Comment by Golgrax** [comment]
> The songs folder is in your Osu's directory, not in the bot's directory. It's where the game stores the beatmaps

yeah, im just showcasing that its running, what my problem is theres no Osu songs folder for linux (i know there is but other name ig)

**Comment by Golgrax** [comment]
well im new to the game and will check it tomorrow when im free, also Happy New Year!

**Comment by vicb0** [comment]
I think Osu has an option in its settings that opens the file explorer in the Beatmaps folder,

Thank you! Happy New Year!
